docker/ray-ml/Dockerfile

```diff
@@ -1,22 +1,10 @@
 ARG BASE_IMAGE
-FROM rayproject/ray:nightly"$BASE_IMAGE"
+ARG FULL_BASE_IMAGE=rayproject/ray:nightly"$BASE_IMAGE"
+FROM "$FULL_BASE_IMAGE"
 
-# We have to uninstall wrapt this way for Tensorflow compatibility
-COPY requirements.txt ./
-COPY requirements_compiled.txt ./
-COPY dl-cpu-requirements.txt ./
-COPY dl-gpu-requirements.txt ./
-COPY ray-docker-requirements.txt ./
-COPY core-requirements.txt ./
-COPY data-requirements.txt ./
-COPY rllib-requirements.txt ./
-COPY rllib-test-requirements.txt ./
-COPY tune-requirements.txt ./
-COPY tune-test-requirements.txt ./
-COPY train-requirements.txt ./
-COPY train-test-requirements.txt ./
-
-COPY install-ml-docker-requirements.sh ./
+COPY *requirements.txt python/*requirements.txt python/requirements/ml/*requirements.txt python/requirements/docker/*requirements.txt ./
```

Comment:
they have to be in one line; otherwise build-docker-images.py will complain that python/ directory doesn't exist

Reply:
could you add this in the comment?

and is it possible to break this into shorter lines with `\` ? like will this still work?

````
COPY \
  *requirements.txt \
  python/*requirements.txt \
  python/requirements/ml/*requirements.txt \ 
  python/requirements/docker/*requirements.txt \
  ./
````